kite_test.go

```diff
@@ -139,6 +145,108 @@ func TestConcurrency(t *testing.T) {
 	wg.Wait()
 }
 
+func TestNoConcurrentCallbacks(t *testing.T) {
+	const timeout = 5 * time.Second
```

Comment:
> your max rand sleep time( 99 )ms * 50 (op count) < 5 secs

Oh nice catch! But it's coincidence - timeout is used per element not globally. I've changed it to 2 seconds.